src/quickgui/plugin/qgsquickfeatureform.qml

```diff
@@ -280,7 +280,7 @@ Item {
       Label {
         id: fieldLabel
 
-        text: qsTr(Name) || ''
+        text: Alias || qsTr(Name) || ""
```

Comment:
why name should be translated here? as it is dynamic variable it does not make much sense